src/common/TracepointProvider.cc

```diff
@@ -40,6 +41,8 @@ void TracepointProvider::verify_config(const ConfigProxy& conf) {
   }
 
   m_handle = dlopen(m_library.c_str(), RTLD_NOW | RTLD_NODELETE);
+  lderr(m_cct) << __func__ << " failed dlopen(): "<< m_library.c_str() <<", "
```

Comment:
This should be conditional on the failure of `dlopen()`. Now we're showing the error always.